src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js

```diff
@@ -272,14 +272,16 @@ class EntityReferenceAutocomplete extends React.Component {
     );
   };
 
-  renderInput = ({ InputProps, ref, label, ...other }) => (
+  renderInput = ({ InputProps, ref, label, autoComplete, ...other }) => (
```

Comment:
Do you think there is ever a usecase where we want the browser based autosuggestion? I would argue that we don't have to make this configurable. 🤔

Reply:
Hi @dawehner 
I was also not so sure about it, so I added a generic case having `autoComplete` default value as `off`

Should we always keep `autocomplete` as `off`?

Reply:
Yeah, I think the autocompletion by the browser is always annoying 👍

Reply:
I think we should force keep it always off 👍